lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -183,9 +182,10 @@
 
                     // Navigate to the default page if there is no history:
                     // 1. If the user belongs to at least one team, jump to the "My Teams" page
-                    // 2. If not, then jump to the "Browse" page
+                    // 2. If the user will not see a "Browse" page, jump to the "My Teams" page
+                    // 3. Otherwise, jump to the "Browse" page
                     if (Backbone.history.getFragment() === '') {
-                        if (this.myTeamsCollection.length > 0) {
+                        if (this.myTeamsCollection.length > 0 || !this.canViewBrowseTab()) {
```

Comment:
This could be simplified if you felt so inclined
```suggestion
                        if (this.myTeamsCollection || !this.canViewBrowseTab()) {
```
-- or --
```suggestion
                        if (this.myTeamsCollection.length || !this.canViewBrowseTab()) {
```

Reply:
Whoa, I've never seen these github "suggestions", that's very neat! Also, sounds good.